trustpinning/certs_test.go

```diff
@@ -340,6 +341,7 @@ func TestValidateRootWithPinnerCertAndIntermediates(t *testing.T) {
 	err = signed.Sign(cs, signedRoot, []data.PublicKey{ecdsax509Key}, 1, nil)
 	require.NoError(t, err)
 
+	signedRoot.Signatures[0].IsValid = true
```

Comment:
I think this is due to the equality checks `require.Equal(t, typedSignedRoot, validatedRoot)`